components/index.js

```diff
@@ -9,6 +9,7 @@ export { default as HighlightedText } from './highlighted-text'
 export { default as Input } from './input'
 export { default as Logo } from './logo'
 export { default as Page } from './page'
+export { default as RowContainer } from './row-container'
```

Comment:
The only reusable piece is the container for images. This is a simple flex box container.